Template/Tag/AddThisTag.web.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var siteId = parameters.get('AddThisPubId');
+            if (siteId) {
+                var script = document.createElement('script');
+                var s = document.getElementsByTagName('script')[0];
+                script.type = 'text/javascript';
+                script.src = '//s7.addthis.com/js/300/addthis_widget.js#pubid=' + siteId;
+                s.parentNode.insertBefore(script, s);
+
+                var parentSelector = parameters.get('AddThisParentSelector');
+                if (parentSelector) {
+                    var parent = TagManager.dom.bySelector(parentSelector)[0];
```

Comment:
same here, we should make sure that a resultSet was found before accessing `[]`. 

Same above  in `document.getElementsByTagName('script')[0]` where we maybe need to fallback to `document.body`? Although now that I think about it: This check here may not be needed as the script element for the tag manager should exist anyway and it should always find one script element.

Reply:
Fixed selector in https://github.com/matomo-org/tag-manager/pull/59/commits/96f6cedcf592543699246f2ea9572b86b630c3d1